napari/plugins/hook_specifications.py

```diff
@@ -1,45 +1,88 @@
 """
-All hook specifications for pluggable functionality should be defined here.
+All napari hook specifications for pluggable functionality are defined here.
 
-A hook specification is a function signature (with documentation) that declares
-an API that plugin developers must adhere to when providing hook
-implementations.  Hook implementations provided by plugins (and internally by
+A *hook specification* is a function signature (with documentation) that
+declares an API that plugin developers must adhere to when providing hook
+implementations.  *Hook implementations* provided by plugins (and internally by
 napari) will then be invoked in various places throughout the code base.
 
-NOTE: in pluggy documentation, hook specification marker instances are named
-"hookspec" by convention, and hook implementation marker instances are named
-"hookimpl".  The convention in napari is to name them more explicity:
-"napari_hook_specification" and "napari_hook_implementation", respectively.
+When implementing a hook specification, pay particular attention to the number
+and types of the arguments in the specification signature, as well as the
+expected return type.
 
-hook specifications are a feature of pluggy:
-https://pluggy.readthedocs.io/en/latest/#specs
+To allow for hook specifications to evolve over the lifetime of napari,
+hook implementations may accept *less* arguments then defined in the
```

Comment:
```suggestion
hook implementations may accept *fewer* arguments than defined in the
```

https://www.youtube.com/watch?v=qmVnr7rsWrE

😁